components/chainhook-sdk/src/observer/mod.rs

```diff
@@ -1142,6 +1236,67 @@ pub async fn start_observer_commands_handler(
                         stacks_chainhooks.len()
                     )
                 });
+                // track stacks chain metrics
+                match &chain_event {
+                    StacksChainEvent::ChainUpdatedWithBlocks(update) => {
+                        match update
+                            .new_blocks
+                            .iter()
+                            .max_by_key(|b| b.block.block_identifier.index)
+                        {
+                            Some(highest_tip_update) => match observer_metrics.write() {
+                                Ok(mut metrics) => {
+                                    if highest_tip_update.block.block_identifier.index
+                                        > metrics.stacks.tip_height
+                                    {
+                                        metrics.stacks.tip_height =
+                                            highest_tip_update.block.block_identifier.index;
+                                    }
+                                    metrics.stacks.last_block_ingestion_at = SystemTime::now()
+                                        .duration_since(UNIX_EPOCH)
+                                        .expect("Could not get current time in ms")
+                                        .as_millis()
+                                        .into();
+                                }
+                                Err(e) => ctx.try_log(|logger| {
+                                    slog::warn!(
+                                        logger,
+                                        "unable to acquire observer_metrics_rw_lock:{}",
+                                        e
+                                    )
+                                }),
+                            },
+                            None => {}
+                        }
+                    }
+                    StacksChainEvent::ChainUpdatedWithReorg(update) => {
+                        // todo: I don't know which block's timestamp we actually want
```

Comment:
@lgalabru 
Another thing I wanted your confirmation on: for reorg time, do we want the timestamp of the highest-tipped new block we're applying, or do we just want `time.now()`